ssas/service/admin/router.go

```diff
@@ -16,17 +18,40 @@ import (
 
 var infoMap map[string][]string
 var adminSigningKeyPath string
+var adminSigningKey string
 var server *service.Server
 
 func init() {
 	infoMap = make(map[string][]string)
 	adminSigningKeyPath = os.Getenv("SSAS_ADMIN_SIGNING_KEY_PATH")
+	adminSigningKey = os.Getenv("SSAS_ADMIN_SIGNING_KEY")
 }
 
 // Server creates an SSAS admin server
 func Server() *service.Server {
 	unsafeMode := os.Getenv("HTTP_ONLY") == "true"
-	server = service.NewServer("admin", ":3004", constants.Version, infoMap, routes(), unsafeMode, adminSigningKeyPath, 20*time.Minute, "")
+
+	var signingKey *rsa.PrivateKey
+
+	if adminSigningKey != "" {
```

Comment:
To reduce the possibility of confusion, should we enforce that only ONE of either the key or the path can be set?  In the current implementation, both values could be set (to potentially different values), but the code will only pick up the key value and ignore the path value.  Should it be considered a server init/startup error if both are set? 
(The same comment applies in `public/router.go`)

Reply:
I was planning on using  the inline environment variable as an override but this is reasonable. I fixed it so that it will fail if both or none are set, and will only accept one or the other.